MbDotNet.Tests/Acceptance/ConfigTests.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace MbDotNet.Tests.Acceptance
+{
+	[TestClass, TestCategory("Acceptance")]
+    public class ConfigTests : AcceptanceTestBase
+    {
+		[TestInitialize]
+		public async Task TestInitialize()
+		{
+			await _client.DeleteAllImpostersAsync();
+		}
+
+        [TestMethod]
+        public async Task GetConfig()
+        {
+            var result = await _client.GetConfigAsync();
+            Assert.IsNotNull(result);
+        }
+
+        [TestMethod]
+        public async Task GetConfig_Version()
+        {
+            const string version = "2.6.0";
+            var result = await _client.GetConfigAsync();
+            Assert.AreEqual(version, result.Version);
+        }
+
+        [TestMethod]
+        public async Task GetConfig_Options()
+        {
+            const bool mock = true;
+            var result = await _client.GetConfigAsync();
+            Assert.AreEqual(mock, result.Options["mock"]);
+        }
+
+        [TestMethod]
+        public async Task GetConfig_Process()
+        {
+            const string nodeversion = "v14.17.2";
+            var result = await _client.GetConfigAsync();
+            Assert.AreEqual(nodeversion, result.Process["nodeVersion"]);
+        }
```

Comment:
These tests will pass/fail based on the configuration of the system that runs them so they are not reliable and should be removed. You can add additional asserts to the remaining test to verify that the `Options` and `Process` collections are not empty.

Reply:
You can see an example of this in the failing CI build - 

```
  X GetConfig_Process [19ms]
  Error Message:
   Assert.AreEqual failed. Expected:<v14.17.2>. Actual:<v16.15.1>. 
  Stack Trace:
     at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid3[T0,T1,T2](CallSite site, T0 arg0, T1 arg1, T2 arg2)
   at MbDotNet.Tests.Acceptance.ConfigTests.GetConfig_Process() in /home/runner/work/MbDotNet/MbDotNet/MbDotNet.Tests/Acceptance/ConfigTests.cs:line 45
```